prov/efa/src/rdm/efa_rdm_msg.c

```diff
@@ -812,6 +813,14 @@ struct efa_rdm_ope *efa_rdm_msg_alloc_rxe_for_msgrtm(struct efa_rdm_ep *ep,
 			return NULL;
 		}
 		(*pkt_entry_ptr)->ope = rxe;
+
+		peer_rxe->size = data_size;
```

Comment:
This is non-tagged msg path. You should implement the change in efa_rdm_msg_alloc_rxe_for_tagrtm

Reply:
@shijin-aws Sorry about that! Updated.